### PR TITLE
Project bmi qurat

### PR DESCRIPTION
**Pull request by qurat-azim**
function code added

### CODE REVIEW

File: tests/testthat/test-project_bmi.R [hunk 1]
```diff
@@ -0,0 +1,23 @@
+test_that("function does not return the bmi trajectory correctly", {
+  fig <- project_bmi(108, 1.87, 27, 60, return_graph=TRUE)
+  expect_true(class(fig)[2] == "htmlwidget")
+  expect_false(is.null(fig$x$attrs))
+  expect_false(is.null(fig$x$layout))
+})
+
+test_that("function does not handle exceptions properly", {
```
Comment (netsgnut) [inline tests/testthat/test-project_bmi.R:8]: Just a nitpick here, the `test_that` names should be a positive statement or expected behavior, like:

`test_that("function handles exceptions properly")`

I have cross-checked the official documentations from `test_that` and it does not really have mentioned this explicitly, but this is how the author wrote the tests in the `testthat` library:
https://github.com/r-lib/testthat/blob/main/tests/testthat/test-examples.R#L1

Comment (qurat-azim) [reply tests/testthat/test-project_bmi.R:8]: i see, I actually told @hcwang24 otherwise.My bad! I'll fix it

Comment (netsgnut) [reply tests/testthat/test-project_bmi.R]: No problem - I have closed that PR too (IMHO I think they should have made this more explicit in their docs too)

I believe the idea is that when using an expectation statement, those lines would read like this:
- `test that it does A, B, and C`
- `test that when A, it throws error`, etc

This is similar to [another testing library](https://rspec.info/documentation/3.12/rspec-expectations/#basic-usage) (in Ruby, though), that the expectation line can be read as something like:
- `it sums the prices of the items in its line items`

**Review by netsgnut — APPROVED** [review]
LGTM :+1: 
Merging

**Pull request merged**